Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace.package]
 version = "1.0.3"
 edition = "2021"
-rust-version = "1.79"
+rust-version = "1.80.0"
```

Comment:
Needs also update in README.md, please also look for all other `1.79` in the repo and update them